developers.diem.com/docs/governance.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: governance
+title: Governance
+---
+
+The Diem Association is responsible for the governance of the Diem network and the development of the Diem project. The Association Council functions as a general assembly of Association members, with each represented by an individual representative.
+
+
+## Diem Improvement Proposals (DIP)
+
+Diem Improvement Proposals (DIPs) offer the Diem developer community a way to participate in advancing the features and functioning of the Diem payment network (DPN), either by proposing a change or engaging in discussion about a proposal. Learn more about how DIPs work [here](https://dip.diem.com/overview). You can also view the current list of [all DIPs]((https://dip.diem.com/all-dips)).
+
+## Technical Steering Committee
+
+The [Technical Steering Committee (TSC)](https://www.diem.com/en-us/about-us/#technical_steering_committee) comprises members who supervise technical design and development of the Diem Blockchain.
+
+Appointed by the Council, the TSC is comprised of representatives from Member organizations who are tasked with advising on the technical design and development of the Diem network.
+
```

Comment:
Should we avoid duplicating content here and just leave the reference to the TSC page?